tbx/project_styleguide/templates/patterns/navigation/components/primary-nav.html

```diff
@@ -1,14 +1,21 @@
 {% load wagtailcore_tags %}
 
 {% for item in primarynav %}
-    {% with link=item.value %}
-        <li>
-            <a href="{{ link.url }}" {% if link.is_page %}data-menu-item{% else %}target="_blank" rel="nofollow noreferrer"{% endif %}>
-                <span>{{ link.text }}</span>
-                {% if not link.is_page %}
-                    {% include "patterns/atoms/icons/icon.html" with name="external" classname="nav-item__external-link-icon" %}
+    {% with children=item.value.page.get_children.live.public.in_menu %}
+        <li class="primary-nav__item {% if children %}primary-nav__item--is-parent{% endif %}"
+            {% if children %}data-has-subnav{% endif %}
+        >
+            {% with link=item.value %}
+                <a href="{{ link.url }}" {% if link.is_page %}data-menu-item{% else %}target="_blank" rel="nofollow noreferrer"{% endif %}>
+                    <span><strong>{{ link.text }}</strong></span>
+                    {% if not link.is_page %}
+                        {% include "patterns/atoms/icons/icon.html" with name="external" classname="nav-item__external-link-icon" %}
+                    {% endif %}
+                </a>
+                {% if children %}
+                    {% include "patterns/navigation/components/includes/subnav-menu-item.html" with links=children %}
```

Comment:
Does this line need updating to filter by whether the pages are published and have 'show in menu' ticked?

Reply:
I think we won't need to, since this is handled here:

https://github.com/torchbox/torchbox.com/blob/a432c3ff491011d0739ca4823b6aee89f31f862a/tbx/project_styleguide/templates/patterns/navigation/components/primary-nav.html#L4

Reply:
ah you're right - I did check one historical work page where it wasn't ticked, and assumed it was typical - but it wasn't!